app/repositories/submission_repository.rb

```diff
@@ -23,28 +31,38 @@ def waitlist
     rated_scope.offset(Setting.get.available_spots).to_a
   end
 
-  def next_to_rate(current_created_at)
-    to_rate_scope.where('submissions.created_at > ?', current_created_at).order('created_at ASC')
-      .first || to_rate_scope.first
+  def unaccepted
+    with_rates_if_any.having('avg(value) < ?', Setting.get.waitlist_threshold).to_a + rejected
+  end
+
+  private
+
+  def get_next_submission(current_created_at)
+    valid.where('submissions.created_at > ?', current_created_at).order('created_at ASC').first
```

Comment:
It used to scroll only through `to rate` submissions, now it scrolls through all of the valid ones, not sure what's the desired behavior